### ISSUE TITLE
batch normalization initialization

### ISSUE THREAD

**Issue by eambutu** [chenxi116/TF-deeplab#3]
In the model, you initialize factor as:

`      factor = tf.get_variable(
          'factor', 1, tf.float32,
          initializer=tf.constant_initializer(0.0, tf.float32),
          trainable=False)`

which makes everything `nan` on first pass

**Comment by chenxi116** [comment]
Are you suggesting the initial value be set to 1.0 instead? 
I always use pre-trained weights, which is probably why I did not pay any attention to the default value.

**Comment by eambutu** [comment]
yup, that would make more sense

**Comment by chenxi116** [comment]
Fixed in https://github.com/chenxi116/TF-deeplab/commit/797f65405b08fdd092a37acc153ab041694923f4
